src/tests/system/tests/test_netgroups.py

```diff
@@ -241,3 +243,121 @@ def test_netgroup__lookup_nested_groups_with_host_and_domain_values_present(
     result = client.tools.getent.netgroup("nested_group")
     assert result is not None
     assert expected in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=802207)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__fully_qualified_names(client: Client, provider: GenericProvider):
+    """
+    :title: Netgroups with fully qualified names resolves and contains the members
+    :setup:
+        1. Configure SSSD domain to use fully qualified names
+        2. Create user "user-1"
+        3. A netgroup named "ng-1" is created, "user-1" is added to this netgroup
+    :steps:
+        1. Verify the existence and membership of the netgroup "ng-1"
+    :expectedresults:
+        1. SSSD should return netgroup "ng-1" and members of the netgroup
+    :customerscenario: True
+    """
+    client.sssd.dom("test")["use_fully_qualified_names"] = "true"
+    user = provider.user("user-1").add()
+    provider.netgroup("ng-1").add().add_member(user=user)
+    client.sssd.start()
+
+    result = client.tools.getent.netgroup("ng-1")
+    assert result is not None and result.name == "ng-1", "'ng-1' Netgroup name did not match '{result.name}'"
+    assert len(result.members) == 1, "'ng-1' contains more than 1 member!"
+    assert "(-, user-1)" in result.members, "'ng-1' members did not match the expected ones!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=678410)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__entry_negative_timeout(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD should respect entry_negative_timeout value and should not return deleted users after
+            this timeout has expired
+    :setup:
+        1. Create user with password
+        2. Set entry_negative_timeout to 1 second
+    :steps:
+        1. Check user is present and can be used for authentication
+        2. Delete the user
+        3. Ensure user exists in SSSD cache immediately after deletion
+        4. Try ssh authentication for deleted user
+        5. Check after entry_negative_timeout user is no longer present in SSSD cache
+    :expectedresults:
+        1. User can be used for authentication
+        2. User is deleted
+        3. User exists in SSSD cache immediately after deletion
+        4. Fail ssh authentication for deleted user
+        5. After entry_negative_timeout user is no longer present in SSSD cache
+    :customerscenario: True
+    """
+    client.sssd.dom("test")["entry_negative_timeout"] = "1"
```

Comment:
Value 1 might make it flaky if we re not fast enough with the id command. I would put 3 there.